hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestSpaceCurveLayoutOptimization.scala

```diff
@@ -119,106 +140,12 @@ class TestTableLayoutOptimization extends HoodieClientTestBase {
     )
   }
 
-  def assertRowsMatch(one: DataFrame, other: DataFrame) = {
+  private def assertRowsMatch(one: DataFrame, other: DataFrame) = {
     val rows = one.count()
     assert(rows == other.count() && one.intersect(other).count() == rows)
   }
 
-  @Test
```

Comment:
Are these tests obsolete or they should still be kept?

Reply:
Yeah these are superseded by `TestColumnStatsIndex`